src/main/java/gov/nasa/pds/harvest/crawler/ProductProcessor.java

```diff
@@ -27,6 +27,21 @@
 import gov.nasa.pds.harvest.util.xml.XmlDomUtils;
 
 
+/**
+ * <p>Process products (PDS4 XML label files) excluding collections and bundles.</p>
+ *
+ * <p> Processing steps:
```

Comment:
Hi @tdddblog 
When I run `mvn javadoc:jar` I have errors :
[ERROR] Exit code: 1 - javadoc: error - The code being documented uses modules but the packages defined in https://docs.oracle.com/javase/8/docs/api/ are in the unnamed module.
[ERROR] /Users/loubrieu/git/harvest/src/main/java/gov/nasa/pds/harvest/crawler/ProductProcessor.java:41: error: unexpected end tag: </p>
[ERROR]  * </p> 
[ERROR]    ^
[ERROR] /Users/loubrieu/git/harvest/src/main/java/gov/nasa/pds/harvest/crawler/CollectionProcessor.java:45: error: unexpected end tag: </p>
[ERROR]  * </p>
[ERROR]    ^
[ERROR] /Users/loubrieu/git/harvest/src/main/java/gov/nasa/pds/harvest/crawler/BundleProcessor.java:44: error: unexpected end tag: </p>
[ERROR]  * </p> 
[ERROR]    ^
[ERROR] /Users/loubrieu/git/harvest/src/main/java/gov/nasa/pds/harvest/crawler/DirsProcessor.java:44: error: unexpected end tag: </p>
[ERROR]  * </p> 

The </p> errors can be corrected by having it moved to line 33 (in this example).

I don't know about the other error (first line)...

Reply:
I fixed an error and warnings.